.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
In the current configuration, it seems that we will publish the unstable chart based on the `main` branch in every commits. In other words, users can use unstable (development) version chart.

Also, the `helm install` command uses the latest version (unstable/development version in this case) if we don't specify the `--version` flag as follows.
```console
$ helm install foo-release scalarflow/api -f ./custom-values-file.yaml
```

So, there is a possibility that users use unstable versions in a production environment by accident. This is my concern.

If developers need to use the latest unstable version, there is another way than publish the latest chart. You can use it as follows.
```console
$ git clone https://github.com/scalar-labs/scalarflow-helm-charts.git
$ cd scalarflow-helm-charts
$ helm install foo-api-release ./scalarflow/charts/api/ -f custom_values_file.yaml
```

So, I think it would be better to don't publish an unstable version chart to users if there is no other special reason.

What do you think?

Reply:
Thank you for pointing it out, I think it is good to not publish the unstable version. I have updated to remove releasing charts for commit in main branch